src/bentoml/_internal/service/service.py

```diff
@@ -30,12 +31,13 @@
     from ..bento import Bento
     from ...grpc.v1 import service_pb2_grpc as services
     from .openapi.specification import OpenAPISpecification
-
 else:
     grpc, _ = import_grpc()
 
 logger = logging.getLogger(__name__)
 
+SerializationStrategy = t.Literal["EXPORT_BENTO", "LOCAL_BENTO", "REMOTE_BENTO"]
```

Comment:
If this is only a type, then we should move `t.Literal` under TYPE_CHECKING block. 

`t.Literal` is not available in 3.7 and it should always live under `t.TYPE_CHECKING`.